model/riscv_insts_base.sail

```diff
@@ -710,6 +749,7 @@ function clause execute WFI() =
 mapping clause assembly = WFI() <-> "wfi"
 
 /* ****************************************************************** */
+$[name "store fence (virtual memory address)"]
```

Comment:
The S is for supervisor not store; it applies to all memory accesses, whether loads, stores or instruction fetches